src/index.js

```diff
@@ -1 +1,105 @@
+/* eslint-disable no-console */
+
 'use strict';
+
+const http = require('http');
+
+const fs = require('fs');
+
+const path = require('path');
+
+const formidable = require('formidable');
+
+const PORT = process.env.PORT || 3000;
+
+const server = new http.Server();
```

Comment:
don't see any sense in adding all these spaces as it makes a code larger
You can use spaces for making code cleaner, but not need to add these too much
```suggestion
const http = require('http');
const fs = require('fs');
const path = require('path');
const formidable = require('formidable');

const PORT = process.env.PORT || 3000;
const server = new http.Server();
```